jscs.json

```diff
@@ -0,0 +1,9 @@
+{
+  "disallowNewlineBeforeBlockStatements": true,
+  "requireBlocksOnNewline": true,
+  "requireCapitalizedComments": true,
```

Comment:
I'm strongly indifferent to this rule.
If I try adding this module as a dependency of something like **gulp-util**, I currently get about 18 code style errors (13 of which are this rule).